braid/braid.c

```diff
@@ -474,6 +475,7 @@ braid_Drive(braid_Core  core)
    _braid_CycleState  cycle;
    braid_Int          iter, level, done, refined;
 
```

Comment:
@jcc242 Can we remove this line?

Reply:
Fixed in cbf1bb00ba48443a77aec25da2d16f2a2007f874